kind.yaml

```
@@ -12,23 +12,24 @@ environment:
       - build-base
       - busybox
       - ca-certificates-bundle
-      - go
+      - go~1.22
```

why do we need to pin to specific version of go 
```suggestion
      - go
```

---

Removing specific version, cleaning this up